packages/next/client/page-loader.js

```diff
@@ -107,30 +124,32 @@ export default class PageLoader {
     })
   }
 
-  async loadScript (route) {
+  async loadRouteBundle (route) {
```

Comment:
Renaming this method is probably going to add extra bytes, how about just `loadRoute`?

Also, I'm not sure if this is a public API that can't have its name changed -- cc @timneutkens.